cmd/credential.go

```diff
@@ -30,12 +30,13 @@ const (
 	accessKeyMinLen = 5
 	accessKeyMaxLen = 20
 	secretKeyMinLen = 8
-	secretKeyMaxLen = 100
 
 	alphaNumericTable    = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 	alphaNumericTableLen = byte(len(alphaNumericTable))
 )
 
+var secretKeyMaxLen = 40
```

Comment:
>keeping different constants creates problem as the signature verification code is the same. we'll solve this differently as there is a refactor planned, not sure of the complete picture yet.

I think I wasn't clear. I am suggesting we use different constants like,
```go
const  (
  globalAzureMaxSecretKeyLen = 100
  globalS3MaxSecretKeyLen = 40
)
```
instead of using the integer literals `40` and `100` directly. 

You can still use `secretMaxKeyLen` with or without global prefixed to it, which will be used by the signature code.